R/cor_test_bayes.R

```diff
@@ -1,6 +1,6 @@
 #' @importFrom stats complete.cases rnorm qnorm
 #' @importFrom utils install.packages
-#' @importFrom effectsize ranktransform
+#' @importFrom datawizard ranktransform
```

Comment:
You don't need to import the functions here because you qualify them with the namespace below.

Reply:
Hmm, I know that we are doing this across `easystats`, but the only hesitation I have against doing so is that usually looking at `NAMESPACE` is how I figure out on which packages I am relying.

For example, let's say the only function I was using from `datawizard` was `ranktransform`, and I refactor the code to no longer rely on it. But I forget to remove `datawizard`  from `Depends`. 

Will `R CMD check` catch this? Does it check for relying on packages which are not used anywhere in the codebase?

Reply:
Ah, if that's your concern, here is a lower-labor way to do it:

```r
find_unused_dependencies <- function(
  dependencies = c("Depends", "Imports", "LinkingTo"),
  include_supp_files = FALSE,
  path = getwd(),
                                     root = NULL,
                                     ...,
                                     progress = TRUE,
                                     errors = c("reported", "fatal", "ignored"),
                                     dev = FALSE
                                     ) {
  deps <- renv::dependencies(path = path, root = root, ...,
                             progress = progress, errors = errors, dev = dev)
  if (isFALSE(include_supp_files)) {
    deps <- dplyr::filter(
      deps, 
      stringr::str_detect(Source, "DESCRIPTION") |
        stringr::str_detect(Source, "/R/")
    )
  }
  deps <- dplyr::filter(deps, Package != "R")
  deps_desc <- data.frame(Package = remotes::local_package_deps(
    pkgdir = path, 
    dependencies = dependencies
  ))
  deps_used <- dplyr::filter(deps, ! stringr::str_detect(Source, "DESCRIPTION"))
  deps_orphan <- dplyr::anti_join(deps_desc, deps_used, by = "Package")
  return(deps_orphan)
}
```

Reply:
```r
find_unused_dependencies()
# Finding R package dependencies ... Done!
#   Package
# 1 poorman
```

Reply:
`dependencies` says which dependencies fields to check. See `?remotes::local_package_deps`.
`include_supp_files`: If `FALSE`, only check for dependencies in `DESCRIPTION` and files in `/R/` (excluding tests, vignettes, `/WIP/`, README, etc.). If `TRUE`, all files are checked.

Reply:
Awesome! Thanks a lot.